src/julia.h

```diff
@@ -397,6 +397,7 @@ typedef struct {
     unsigned exportp:1;
     unsigned imported:1;
     unsigned deprecated:1;
+    jl_sym_t *deprecation_preferred_name;  // non-NULL when new type incompatible with old
```

Comment:
I see what you mean. I can fix that.

But I can't tell, how big a deal is it to increase the size by one pointer? That's this issue this is stuck on. I'll set the "decision" flag on this, hoping for a clear verdict one way or another.